.github/workflows/CMake-minimal-version.yaml

```diff
@@ -0,0 +1,48 @@
+name: CMake minimal version
+# The aim of this test derminate minimal supported CMake version for LibRS project
+# and check if it equlas to expected minimal CMAke version
+
+on:
+  push:
+  pull_request:
+  
+env:
+  EXPECTED_CMAKE_VERSION: "VERSION 3.8.0"
+
+jobs:
+  test_downloader:
+    name: "Test cmake_downloader.py"
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: "Get current CMake version"
+        id: cmake_version
+        run: |
+          sudo apt-get install  xorg-dev libglu1-mesa-dev
+          
+          cd ..
+          git clone https://github.com/nlohmann/cmake_min_version.git --depth 1
```

Comment:
Add comment that we currently take the latest commit, if we see issues in the future, we will stabilize on this commit which worked: `687dc56e1cf52c865cebf6ac94ad67906d6e1369`